### PR TITLE
feat(groups): adds keyboard bind to delete a group

### PR DESCRIPTION
**Pull request by 2xAA**

### CODE REVIEW

File: src/components/Group.vue [hunk 1]
```diff
@@ -530,12 +511,20 @@ export default {
...
     hasLink(id) {
       return this.$modV.store.state.inputs.inputLinks[id];
     },
 
     isFocused(id) {
       return this.$modV.store.state.inputs.focusedInput.id === id;
+    },
+
+    removeGroup(e) {
+      console.log(e.keyCode);
```
Comment (TimPietrusky) [inline src/components/Group.vue]: Can we get rid of this?

File: src/components/Group.vue [hunk 2]
```diff
@@ -7,9 +7,8 @@
       type: 'Group',
       focusElement: true
     }"
-    @focus="focus"
```
Comment (TimPietrusky) [inline src/components/Group.vue:10]: Why are you removing the focus stuff in this PR?

File: src/components/Group.vue [hunk 3]
```diff
@@ -7,9 +7,8 @@
...
-    @mousedown="focus"
-    :class="{ focused }"
+    tabindex="0"
```
Comment (TimPietrusky) [inline src/components/Group.vue:10]: Why is the tabindex always 0?

**Review by TimPietrusky — CHANGES_REQUESTED** [review]
@2xAA I have a few questions!

File: src/components/Group.vue (hunk 3 above)
Comment (2xAA) [reply src/components/Group.vue:10]: > tabindex="0" allows elements besides links and form elements to receive keyboard focus. It does not change the tab order, but places the element in the logical navigation flow, as if it were a link/button on the page.

https://webaim.org/techniques/keyboard/tabindex

Element focus is required to allow the keydown event to work.

File: src/components/Group.vue (hunk 2 above)
Comment (2xAA) [reply src/components/Group.vue:10]: It was previously used for selecting the group you wanted to delete, however it was custom functionality and deletion wasn't possible.

It's no longer needed with the changes in this PR as it isn't used anywhere else.

File: src/components/Group.vue (hunk 1 above)
Comment (2xAA) [reply src/components/Group.vue]: Yes

File: src/components/Group.vue (hunk 3 above)
Comment (TimPietrusky) [reply src/components/Group.vue:10]: Thank you, was not aware of this difference!

**Review by TimPietrusky — APPROVED** [review]
LGTM and was tested!

**Pull request merged**